src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -447,14 +447,19 @@ public static EnabledValue ParseEnableValue(string value)
             {
                 return EnabledValue.ExplicitOptOut;
             }
-            return EnabledValue.Undefined;
+
+            string messageText = string.Format(Strings.Error_InvalidNuGetAuditValue, value, "true, false");
+            RestoreLogMessage message = RestoreLogMessage.CreateError(NuGetLogCode.NU1014, messageText);
```

Comment:
I'm having second thoughts about invalid values for `NuGetAudit`, `NuGetAuditLevel` and `NuGetAuditMode` causing a restore error. Does anyone else think it should be just a warning instead, so customers can NoWarn it if they really want to keep the invalid value and use default behaviour?

Reply:
In a lot of other features, bad values are even silently ignored. I don't mind this. 
Worst case scenario, people run into issues when using newer tooling and older tooling combiantions because the newer would recognize newer options and the older one wouldn't, but we've had discussions that alerting the users of such discrepancies might be ok in some cases.

Reply:
I really think it's bad UX silently ignoring bad inputs. How many issues in Home are from customers trying to do something, it not working as they expected, and there's zero feedback about why? It doesn't guide customers to the path of success.

I'll change this to a warning, so unfortunately I'll need another approval 😞